packages/syntax-core/src/RadioButton/RadioButton.module.css

```diff
@@ -0,0 +1,76 @@
+.baseRadioButton {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  cursor: pointer;
+}
+
+.smBase {
+  height: 16px;
+}
+
+.mdBase {
+  height: 24px;
+  gap: 8px;
```

Comment:
@JFChoe it's odd that we have a `gap` here but not on `smBase`, why is that?

Should we override the `width` / `height` on `radioStyleOverride` depending on the size?